icons/undo-2.json

```diff
@@ -8,7 +8,10 @@
   "tags": [
     "redo",
     "rerun",
-    "history"
+    "history",
+    "backwards",
```

Comment:
`back`, `revert`, `u-turn`

Reply:
> `back`, `revert`, `u-turn`

@jguddas I've been operating under the assumption that e.g a tag of `backwards` would negate the need for the substring `back` as separate tag, in the search…? @karsa-mistmere

Reply:
For searching, yes, although I think actually simply `back` would suffice, since it's the tag that all other left pointing arrow icons have.